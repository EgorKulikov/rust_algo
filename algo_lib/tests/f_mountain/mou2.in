50
I 5 8 0
I 21 24 0
I 25 28 0
Q 0
I 17 20 35
I 23 26 11
I 43 46 -46
Q 184
I 28 31 0
I 40 43 34
I 22 25 40
Q 259
I 15 18 175
I 5 8 0
I 24 27 155
Q 1346
I 29 30 235
I 26 27 373
I 8 9 235
Q 2015
I 1 2 0
I 44 45 159
I 6 7 0
Q 1639
I 28 29 -146
I 25 26 -41
I 8 9 235
Q 160
I 16 17 41
I 33 34 347
I 24 25 -63
Q 1368
I 42 42 489
I 16 16 -15
I 40 40 96
Q 2443
I 20 20 -160
I 41 41 -238
I 30 30 -24
Q 1066
I 18 18 253
I 48 48 216
I 22 22 -8
Q 1568
I 26 26 -123
I 10 10 168
I 28 28 -786
Q 1422
E
