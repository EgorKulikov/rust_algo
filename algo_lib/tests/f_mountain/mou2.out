50
50
23
26
26
25
7
32
41
33
41
44
